app/models/ability.rb

```diff
@@ -196,15 +188,13 @@ def signed_in_with_cfp_role(user)
                          commentable_id: Event.where(program_id: Program.where(conference_id: conf_ids_for_cfp).pluck(:id)).pluck(:id)
 
     # Abilities for Role (Conference resource)
-    can :index, Role
-
-    can :manage, Role do |role|
+    can :read, Role do |role|
       role.resource_type == 'Conference' && (conf_ids_for_cfp.include? role.resource_id)
     end
 
-    # Can add or remove users from role, when user has that same role for the conference
+    # Can add or remove users from role and edit role description, when user has that same role for the conference
     # Eg. If you are member of the CfP team, you can add more CfP team members (add users to the role 'CfP')
-    can :toggle_user, Role do |role|
+    can [:toggle_user, :update, :edit], Role do |role|
       role.resource_type == 'Conference' &&
```

Comment:
I would only keep toggle user here. This is not from a technical perspective, but from an essential one. I'd rather allow only the organizer to change the description of roles in his/her conference.